ui/src/message/Messages.tsx

```diff
@@ -70,7 +70,6 @@ class Messages extends Component<IProps & Stores<'messagesStore' | 'appStore'>,
                         <Button
                             id="refresh-all"
                             variant="contained"
-                            disabled={!hasMessages}
```

Comment:
If the webapp is cache-reopened, then it's likely that the websocket connection isn't reconnected. So you will be able to refresh the messages, but this won't reconnect the websocket.

Reply:
Could we somehow get the refresh button to check connection status and call upon WebSocketStore.listen if needed? I'm not all familiar with how react works.

Reply:
Possible yes, but I'd rather solve this automatically on page load but I currently don't know a way to reproduce this cached reopen where the site state is kept.

Reply:
Assuming your browser reopens previously opened tabs, open gotify in a tab and close the browser. Reopen the browser and thats it. Or open gotify, leave the browser open and sleep the computer.

Reply:
I'm doing this, but my firefox correctly reestablishes the websocket connection and has messages that were sent while the browser was closed.